client/components/drop-zone/index.jsx

```diff
@@ -164,12 +160,6 @@ export class DropZone extends React.Component {
 
 	toggleDropZoneReduxState = ( isVisible ) => {
 		if ( this.state.lastVisibleState !== isVisible ) {
-			if ( isVisible ) {
-				this.props.showDropZone( this.props.dropZoneName );
-			} else {
-				this.props.hideDropZone( this.props.dropZoneName );
-			}
-
```

Comment:
`toggleDropZoneReduxState` can be removed entirely. It now only sets the `lastVisibleState` state which is not read anywhere.

Reply:
Nice find 👍  Further cleaned it up in 24539d42fb66f1af5fad0128b0a1f0d918a0ff0f